VERSIONINFO

```diff
@@ -1,2 +1,2 @@
-appVersion=0.1.1
-version=0.1.1
\ No newline at end of file
+appVersion=0.1.3
```

Comment:
why the appVersion is not 0.1.2?

Reply:
In docker registry, the version 0.1.2 is existing. 
So I don't want to cover it.

Reply:
Done, update appversion to 0.1.2